drivers/clock_control/Kconfig.stm32

```diff
@@ -199,22 +206,37 @@ config CLOCK_STM32_MCO2_SRC_HSE
 	bool "HSE"
 	depends on SOC_SERIES_STM32F4X || \
 	           SOC_SERIES_STM32F7X || \
-	           SOC_SERIES_STM32H7X
+	           SOC_SERIES_STM32H7X || \
+	           SOC_SERIES_STM32H5X
 	help
 	  Use HSE as source of MCO2
 
 config CLOCK_STM32_MCO2_SRC_LSI
 	bool "LSI"
-	depends on SOC_SERIES_STM32H7X
+	depends on SOC_SERIES_STM32H7X || \
+	           SOC_SERIES_STM32H5X
 	help
 	  Use LSI as source of MCO2
 
 config CLOCK_STM32_MCO2_SRC_CSI
 	bool "CSI"
-	depends on SOC_SERIES_STM32H7X
+	depends on SOC_SERIES_STM32H7X || \
+	           SOC_SERIES_STM32H5X
 	help
 	  Use CSI as source of MCO2
 
+config CLOCK_STM32_MCO2_SRC_PLL1P
```

Comment:
This clock source already exist and is named "CLOCK_STM32_MCO2_SRC_PLLPCLK"

Reply:
This should be removed now that you added "CLOCK_STM32_MCO2_SRC_PLLPCLK" for SOC_SERIES_STM32H5 ;)